src/components/RichTextEditor/helpers/exampleValues/exampleValues.ts

```diff
@@ -263,19 +256,19 @@ export const defaultValue = [
             },
         ],
     },
-    createElement({ text: 'Heading 1', element: TextStyles.ELEMENT_HEADING1 }),
-    createElement({ text: 'Heading 2', element: TextStyles.ELEMENT_HEADING2 }),
-    createElement({ text: 'Heading 3', element: TextStyles.ELEMENT_HEADING3 }),
-    createElement({ text: 'Heading 4', element: TextStyles.ELEMENT_HEADING4 }),
-    createElement({ text: 'Custom 1', element: TextStyles.ELEMENT_CUSTOM1 }),
-    createElement({ text: 'Custom 2', element: TextStyles.ELEMENT_CUSTOM2 }),
-    createElement({ text: 'Custom 3', element: TextStyles.ELEMENT_CUSTOM3 }),
-    createElement({ text: 'Quote', element: TextStyles.ELEMENT_QUOTE }),
+    createElement({ text: 'Heading 1', element: 'heading1' }),
```

Comment:
I am not a fan of fixed values, because this are part of the plugins constant. Can for this be introduced a variables on one place and replaced all over.

Reply:
Would not recommend to use `exampleTextStyles` because the name suggest example, should be for the styles. This will need be change again. If it is use for Text style it could stay TextStyle...
The previous style had constants and I dont see the reason why the new style should not have it and was introduced with fixed values which introduces more space for bugs.

Reply:
renamed to `TextStyles`